### DISCUSSION TITLE
new "argon" skin judgements (meh/ok/good/great) look like hit lightning

### DISCUSSION THREAD

**Discussion by Hexality** [ppy/osu#20407] [Ideas] [answered]
is this really supposed to be distracting like that? 
since basically, turning hit lightning off changes almost nothing, still leaving this trail behind like if I were with the hit lightning on.
![image](https://user-images.githubusercontent.com/17398632/191835399-d451773f-d363-44e4-9d9a-b457b19aa514.png)
(based on the old default skin, which when the hit lightning was off, the trail would be basically be gone).

**Comment by Hexality** [comment]
![image](https://user-images.githubusercontent.com/17398632/191836831-d52e7773-ec41-42a5-b068-12e64f703bc5.png)
when enabling hit lightning, the difference is almost imperceptible, the "note trail" just gets a bit brighter.
Idk if its really supposed to be like this as there's some other people thinking just like me about the way that "note trail/judgement" looks.
![image](https://user-images.githubusercontent.com/17398632/191837836-60a85ef0-9e37-4bff-9ca0-49607a99d3d1.png)

**Reply by mk56-spn** [reply]
not every non minimalistic judgement is "hit lighting" as can be seen in many old skins that had much more prominent judgements than this

**Reply by peppy** [reply]
Thanks for feedback. This skin is work in progress and will improve over time.

**Comment by peppy** [comment, marked as answer]
Thanks for feedback. This skin is work in progress and will improve over time.

Hit lighting toggle is not supported yet.
